custom-workload.go

```diff
@@ -36,7 +36,10 @@ func CustomWorkload(wh *workloads.WorkloadHelper) *cobra.Command {
 				log.Fatalf("Error reading custom configuration file: %v", err.Error())
 			}
 			configFileName := strings.Split(configFile, ".")[0]
-			wh.Run(configFileName, nil, nil)
+			os.Unsetenv("ALERTS")
+			os.Unsetenv("METRICS")
+			os.Unsetenv("TS_METRICS")
```

Comment:
Why do you unset these, they are not set anywhere

Reply:
Performing this check only for alerts just in case if alerting is enabled for custom-workload.